ansible/group_vars/sonic/vars

```diff
@@ -1,9 +1,9 @@
 ansible_ssh_user: admin
-ansible_ssh_pass: password
-ansible_become_pass: password
+ansible_ssh_pass: admin
+ansible_become_pass: admin
```

Comment:
I submitted a PR: [https://github.com/Azure/sonic-buildimage/pull/197/files](https://github.com/Azure/sonic-buildimage/pull/197/files). Please wait after that.

---
In reply to: [96720233](https://github.com/Azure/sonic-mgmt/pull/80#discussion_r96720233) [](ancestors = 96720233)